src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCF.java

```diff
@@ -0,0 +1,759 @@
+            final List<VariantContext> truthVariants = new ArrayList<>( nearbyTruths.next().getValue() );
+            for (int i = 0; i < truthVariants.size(); ++i ) {
+
+                final VariantContext truthVarCtx = truthVariants.get(i);
+                if ( ! typesCompatible(eval, truthVarCtx) && bestNearbyTruth == null) {
+                    bestNearbyTruth = new NearbyTruth(truthVarCtx, 0, NearbyTruth.SupportType.INCOMPATIBLE_TYPE); // TODO: 9/6/18 here we have some ambiguity: if none of the nearby truths are of compatible type, this always sets the incompatible nearby truth as the one of the highest coordinate, but we are not currently using this truth downstream, so it's ok for now
+                } else { // at least types are compatible
+                    final float concordanceScore = computeConcordanceScore(eval, truthVarCtx, null);
+
+                    // TODO: 6/3/18 test if heuristic overlap-based concordance score is over a hard threshold, in the long run, haplotype concordance should be evaluated instead
+                    final boolean concordant = recipOvpFracThreshold <= concordanceScore;
+
+                    if (concordant) {
+                        if (concordanceScore > maxConcordanceScore) { // conditionally update to best support
+                            final VariantContextBuilder variantContextBuilder = new VariantContextBuilder(truthVarCtx).rmAttribute(TEMPORARY_CONCORDANCE_STRING_FOR_TRUTH);
+                            final VariantContext updatedTruthVariant = eval.isFiltered() ? variantContextBuilder.attribute(TEMPORARY_CONCORDANCE_STRING_FOR_TRUTH, ConcordanceState.FILTERED_FALSE_NEGATIVE.getAbbreviation()).make()
+                                                                                         : variantContextBuilder.attribute(TEMPORARY_CONCORDANCE_STRING_FOR_TRUTH, ConcordanceState.TRUE_POSITIVE.getAbbreviation()).make();
+                            truthVariants.set(i, updatedTruthVariant);
+
+                            bestNearbyTruth = new NearbyTruth(truthVarCtx, concordanceScore, NearbyTruth.SupportType.GOOD_SUPPORT);
+                            maxConcordanceScore = concordanceScore;
+                        }
+                    } else if (bestNearbyTruth == null || bestNearbyTruth.getSupportType() != NearbyTruth.SupportType.GOOD_SUPPORT){
+                        bestNearbyTruth = new NearbyTruth(truthVarCtx, concordanceScore, NearbyTruth.SupportType.LOW_SUPPORT);
+                    }
+                }
+            }
+        }
+        return bestNearbyTruth;
+    }
+
+    private boolean typesCompatible(final VariantContext eval, final VariantContext truth) {
+        final StructuralVariantType evalType = StructuralVariantType.valueOf(eval.getAttributeAsString(VCFConstants.SVTYPE, ""));
+        final StructuralVariantType truthType = StructuralVariantType.valueOf(truth.getAttributeAsString(VCFConstants.SVTYPE, ""));
+
+        if ( evalType.equals(truthType) ) {
```

Comment:
Maybe this should be encoded in ```SVType``` enum itself?

Reply:
after our discussion, I think this is OK